sagemaker-python-sdk/pytorch_horovod_mpi/pytorch_horovod_mpi.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training PyTorch Models using Horovod with Open MPI"
+   ]
```

Comment:
note which Studio kernel you used to run this, e.g. *(This notebook was tested with the "Python 3 (Data Science)" kernel.)*